### ISSUE TITLE
Small image size leads to wrong format selection in imagesize widget

### ISSUE THREAD

**Issue by koertho** [contao/contao#315]
**Affected version(s)**
Encountered it with 4.4.34

**Description**  
If you set an an image size (for example 6) and you have an predefined image format with the id 6, the imageSize widget will use the last one and also the image format will be used to display the image. 

The error lays in the option "select" handling, where all array values and the current option are compared and select is returned, if one match is found. But this leads to multipe selects, where the last one wins.

https://github.com/contao/contao/blob/a52821a8bb760a2e4a77fe51ac4dc6f294294aed/core-bundle/src/Resources/contao/library/Contao/Widget.php#L1199

**How to reproduce**  
Create an predefined image format and check the id (for example 5).
Edit an image size, set the format to an relative format (for example proportional) and set the width to the id of the predefined image (5 in this example).

**Comment by ausi** [comment]
I can confirm the issue in Contao 4.4.

Contao 4.6 is not affected because we changed the order of the image size groups in d2bf981c03ec133d1fc47544c58bc37bbee40a0b and the later option always wins.

Can be fixed by updating https://github.com/contao/contao/blob/a52821a8bb760a2e4a77fe51ac4dc6f294294aed/core-bundle/src/Resources/contao/widgets/ImageSize.php#L171-L191

```diff
@@ -175 +175 @@ class ImageSize extends Widget
-    $this->isSelected($arrOption),
+    $this->optionSelected($arrOption['value'], $this->varValue[2]),
@@ -186 +186 @@ class ImageSize extends Widget
-    $this->isSelected($arrOptgroup),
+    $this->optionSelected($arrOptgroup['value'], $this->varValue[2]),
```

**Comment by leofeyer** [comment]
Fixed in 5fae7e7ebb1ed0ddf92971dee7057e15029bc533. Thank you @ausi.
